### DISCUSSION TITLE
Using uWebSockets as a Client to Connect to Binance WebSocket API

### DISCUSSION THREAD

**Discussion by bansalshaleen** [uNetworking/uWebSockets#1781] [Q&A] [answered]
Hello,

I am interested in using uWebSockets as a client to connect to a WebSocket and stream data. Specifically, I would like to connect to Binance's top feed WebSocket at the URL wss://stream.binance.com:9443/ws and subscribe to receive BTC top feed data.

Could you please advise on whether this is possible using uWebSockets? If so, are there any examples or documentation that could help me implement this?

Thank you for your assistance.

Best regards,
Shaleen

**Comment by uNetworkingAB** [comment, marked as answer]
It's not possible, uWS has no client only an idea of a portential interface

**Reply by bansalshaleen** [reply]
Thanks for the quick reply.
